### PR TITLE
First test! `meteor npm run chimp-test`

### PR DESCRIPTION
**Pull request by rodrigok**
@RocketChat/core 

This is the first **acceptance test** implementation

Steps:
```
  Basic usage
    ✓ load page (58ms)
    ✓ crate user (6567ms)
    ✓ logout (486ms)
    ✓ login (2090ms)
    ✓ open GENERAL (2250ms)
    ✓ send a message (655ms)
```

# Running tests
Start a Rocket.Chat instance at `localhost:3000` then in another shell:
```
meteor npm run chimp-test
```

### CODE REVIEW

File: tests/test.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+		browser.waitForExist('.main-content', 5000);
+
+		done();
+	});
+
+	it('logout', function(done) {
+		browser.waitForVisible('.account-box');
+		browser.click('.account-box');
+		browser.pause(200);
```
Comment (marceloschmidt) [inline tests/test.js]: Would this .pause() be required, since right after you wait until visible `#logout`?

Comment (rodrigok) [inline tests/test.js]: Yes I guess, cuz the element is moving/animating I think this was causing some test fails

**Comment by marceloschmidt** [comment]
Important to say: `npm install chimp -g` if you haven't done it yet :)

**Comment by sampaiodiego** [comment]
> Important to say: npm install chimp -g if you haven't done it yet :)

it could be added to `devDependencies` to avoid it.

**Comment by rodrigok** [comment]
@sampaiodiego You should have that binary as global I guess.

**Comment by marceloschmidt** [comment]
I'm testing that right now

**Comment by marceloschmidt** [comment]
It works!
I ran `npm remove chimp -g`; then I ran `npm install chimp --save-dev` and tried running `meteor npm run chimp-test` and it worked. Then I deleted the `node_modules` folder to check if running `npm install` would make it work again, and it does.

**Comment by rodrigok** [comment]
@marceloschmidt Great, can you commit?

**Comment by rodrigok** [comment]
@marceloschmidt @sampaiodiego I need a positive review here 😄

**Comment by marceloschmidt** [comment]
Will Travis know how to call our tests? Do we need to change anything there?

**Comment by rodrigok** [comment]
@marceloschmidt I don't we should call this test in Travis now, cuz we need an instance running to test, we should add some unit tests and run they in travis

**Comment by marceloschmidt** [comment]
Gotcha

**Pull request merged**